core/trino-main/src/test/java/io/trino/sql/planner/planprinter/TestJsonRepresentation.java

```diff
@@ -107,7 +108,7 @@ public void testLogicalJsonPlan()
                                         ImmutableList.of("quantity := tpch:quantity"),
                                         ImmutableList.of(new PlanNodeStatsAndCostSummary(60175, 541575, 541575, 0, 0)),
                                         ImmutableList.of())))))));
-        MaterializedResult expectedPlan = resultBuilder(queryRunner.getDefaultSession(), createVarcharType(1896))
+        MaterializedResult expectedPlan = resultBuilder(queryRunner.getDefaultSession(), createVarcharType(1892))
```

Comment:
Why did this change ?

Reply:
I did a minor change to `formatCollection` where now it prints empty string instead of `[]` for empty collection.